src/sentry/search/events/builder/metrics.py

```diff
@@ -105,6 +105,7 @@ def __init__(
         # Dataset.PerformanceMetrics is MEP. TODO: rename Dataset.Metrics to Dataset.ReleaseMetrics or similar
         dataset: Dataset | None = None,
         granularity: int | None = None,
+        time_window: int | None = None,
```

Comment:
let's rename this to `time_range_window` or something like that to indicate it's only ever a relative range. and add some doc strings to explain why we added it and validation (like, if start and end is specified, time_range_window probably shouldn't be etc) better to be restrictive to begin with.